Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxMask/TextBoxMask.cs

```diff
@@ -354,10 +398,10 @@ private static void Textbox_TextChanging(TextBox textbox, TextBoxTextChangingEve
                     i < (oldSelectionStart - deleteBackspaceIndex + singleOrMultiSelectionIndex);
                     i++)
                 {
-                    var maskChar = mask[i];
+                    var maskChar = escapedMask[i];
 
                     // If dynamic character a,9,* or custom
-                    if (representationDictionary.ContainsKey(maskChar))
+                    if (representationDictionary.ContainsKey(maskChar) && !escapedChars.Contains(i))
```

Comment:
Do we need to store the escaped chars/mask? Could we just check escapes when needed and add a check if `i >0` and `textArray[i-1] != '\\'`?

Reply:
@michael-hawker I tried to do that, however, my implementation [was buggy](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/2820/commits/1045b730b675de90e1d84887417f94ce5d90d0c4), I concluded that storing the indices of the escaped chars was the simplest way of doing it.